package.json

```diff
@@ -57,13 +57,16 @@
     "shorthash": "0.0.2",
     "source-map": "^0.7.1",
     "stream-browserify": "^2.0.1",
+    "styled-components": "^4.1.2",
```

Comment:
I don't think we need it in dependencies. It's better to do it the way SassPlugin is done. Require it optionally and throw and error if it's not installed.

Reply:
`styled-components` was needed only for testing purposes, I moved it together with `typescript-plugin-styled-components` to `devDependencies`.